jekyll-build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -euo pipefail
+
+docker build . -f docker/BuildJekyll.Dockerfile --build-arg gid=${SUDO_GID} --build-arg gname=$(getent group $SUDO_GID | cut -d: -f1) --build-arg uid=${SUDO_UID} --build-arg uname=${SUDO_USER} -t jekyll
```

Comment:
I like adding `"$@"` somewhere in a wrapped command like this, so if you write:

```
./jekyll-build.sh --quiet --pull
```

The script will pass those options along to the `docker build` command.